pywbemtools/pywbemcli/_cmd_class.py

```diff
@@ -67,6 +69,22 @@
                       'the returned class(es). '
                       'Default: Include superclass properties and methods.')]
 
+associations_option = [              # pylint: disable=invalid-name
```

Comment:
I suggest to name the variable `association_filter_option` instead, for more clarity on what it is.

Reply:
DONE